src/Components/ImagesTable/ImagesTable.js

```diff
@@ -160,12 +161,41 @@ const ImagesTable = () => {
         <EmptyState variant={EmptyStateVariant.large} data-testid="empty-state">
           <EmptyStateIcon icon={PlusCircleIcon} />
           <Title headingLevel="h4" size="lg">
-            Create an image
+            Create an RPM-DNF image
           </Title>
           <EmptyStateBody>
-            Create OS images for deployment in Amazon Web Services, Microsoft
-            Azure and Google Cloud Platform. Images can include a custom package
-            set and an activation key to automate the registration process.
+            <Text>
+              Image builder is a tool for creating deployment-ready customized
+              system images: installation disks, virtual machines, cloud
+              vendor-specific images, and others. By using image builder, you
+              can create these images faster than manual procedures because it
+              eliminates the specific configurations required for each output
+              type.
+            </Text>
+            <br />
+            <Text>
+              With RPM-DNF, you can manage the system software by using the DNF
+              package manager and updated RPM packages. This is a simple and
+              adaptive method of managing and modifying the system over its
+              lifecycle.
+            </Text>
+            <br />
+            <Text>
```

Comment:
It seems like this `<Text>` component might not be necessary, what do you think?

Reply:
It has effect on the spacing. If we want the double empty space between the link and the "Create image" button, we can add another line break instead of the Text component. Both variants those work for me.

Looks like this without the `<Text>` tags and extra linebreak:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/22cf65e5-e9af-4041-a3d2-0a6c63abd548)

Reply:
Switched the `Text` component for one extra linebreak